cms/src/components/content/EditContentParent.vue

```diff
@@ -56,16 +57,16 @@ const canEdit = computed(() => {
 
         <LInput
             name="parent.image"
-            label="Default image"
+            label="Image"
             :icon="LinkIcon"
             placeholder="https://..."
             :disabled="!canEdit"
             v-model="parent.image"
-        >
-            This image can be overridden in a translation
```

Comment:
Just for my understanding, this was just removed for now because we don't have images yet on the content, but when we have that this will come back, right? Makes sense to remove it now, it's confusing when the other functionality is not there.

Reply:
Correct